Frontend/library/src/PeerConnectionController/PeerConnectionController.ts

```diff
@@ -407,6 +407,26 @@ export class PeerConnectionController {
                             });
                         });
 
+                    // If the user has toggle the "UseFlexFec" flag, we want to add it to the top of the preferences list.
+                    // We don't want it as a selectable codec because the first "actual" codec in the list is the codec that will be
+                    // protected.
+                    if(this.config.isFlagEnabled(Flags.UseFlexFec)) {
+                        const matcher = /(flexfec).*/;
+                        RTCRtpReceiver.getCapabilities('video').codecs.forEach((codec) => {
+                            const str = codec.mimeType.split('/')[1] + ' ' + (codec.sdpFmtpLine || '');
+                            const match = matcher.exec(str);
+                            if (match !== null) {
+                                codecs.push({
+                                    mimeType: codec.mimeType,
+                                    clockRate: codec.clockRate,
+                                    sdpFmtpLine: codec.sdpFmtpLine
+                                        ? codec.sdpFmtpLine
+                                        : ''
+                                });
+                            }
+                        });
+                    }
```

Comment:
Given this whole code path is only evaluated if `transceiver.setCodecPreferences` exists, what happens on FF, does flexfec work?